suites/google/googleDataAccessTest.js

```diff
@@ -75,31 +77,269 @@ BeforeSuite(async (indexd) => {
+  console.log('deleting keys for SA associated with users 0, 1 and user2...');
+  ['user0', 'user1', 'user2'].forEach(async(user) => {
+    const getCredsRes = await fence.do.getUserGoogleCreds(users[user].accessTokenHeader);
+    console.log(`Keys from ${user}: ${JSON.stringify(getCredsRes.access_keys)}`);
+    if (getCredsRes.access_keys.length > 0) {
+      let saName = getCredsRes.access_keys[0].name.split('/')[3];
+      console.log(`delete any existing keys for service account ${saName}`);
+      const dcfSaKeys = await google.listServiceAccountKeys('dcf-integration', saName);
+      console.log(`#### ##:' ${JSON.stringify(dcfSaKeys.keys)}`);
+      if (dcfSaKeys.keys) {
+        dcfSaKeys.keys.forEach(async (key) => {
+          console.log(`the following key will be deleted: ${key.name}`);
+          await google.deleteServiceAccountKey(key.name).then((deletionResult) => {
+            console.log(`deletionResult: ${JSON.stringify(deletionResult)}`);
+            if (deletionResult instanceof Error) {
+              console.log(`WARN: Failed to delete key [${key.name}] from Google service account [${saName}].`);
+	      } else {
+                console.log(`INFO: Successfully deleted key [${key.name}] from Google service account [${saName}].`);
+	      }
+          });
+        })
+      };
+    }
+  });
+});
+
 After(async (fence, users) => {
   // Cleanup after each scenario
-  const unlinkResults = Object.values(users).map((user) => fence.do.unlinkGoogleAcct(user));
+  const unlinkResults = Object.values(users).map(async(user) => {
+    await apiUtil.sleepMS(1 * 1000).then(() => {
+      fence.do.unlinkGoogleAcct(user)
+    });
+  });
   await Promise.all(unlinkResults);
+  console.log('Running usersync job');
+  bash.runJob('usersync', args = 'FORCE true');
 });
 
-Scenario('Test Google Data Access (signed urls and temp creds) @reqGoogle @googleDataAccess',
+Scenario('Test Google Data Access user0 (signed urls and temp creds) @reqGoogle @googleDataAccess',
+  async (fence, indexd, users, google, files) => {
+    console.log(`Double-check if file ${indexed_files.qaFile.did} is indexed. If it isn't fail fast.`);
+    const indexdLookupRes = await indexd.do.getFile(indexed_files.qaFile, users.user0.accessTokenHeader);
+    chai.expect(indexdLookupRes,
+      `First sync: Check if the [indexed_files.qaFile] (${indexed_files.qaFile.filename}) has been indexed. Otherwise fail fast.`).to.have.property('file_name', indexed_files.qaFile.filename);
+
+    let User0signedUrlQA1FileContents = '';
+    let User0signedUrlQA1Res = '';
+    let tempCreds0Res = '';
+
+    // Cannot retry more tha 10 times due to SA key limit (10)
+    // FYI: All Keys are cleaned up in the Before() block
+    const nAttempts = 8;
+    for (let i = 0; i < nAttempts; i += 1) {
+      console.log('make sure google account user0 is unlinked');
+      const unlinkResult = await fence.complete.forceUnlinkGoogleAcct(users.user0);
+      await apiUtil.sleepMS(1 * 1000);
```

Comment:
the above should only return once it's complete, I'm not sure the sleep is absolutely necessary in this case

Reply:
Same situation from this other comment:
https://github.com/uc-cdis/gen3-qa/pull/222/files/f21b4713a85bc1cfd2c2cce16b8dad5da75e6f90#r369252285